### ISSUE TITLE
Problem with custom name of the database

### ISSUE THREAD

**Issue by esiek** [uPhyca/stetho-realm#35]
Hi, 
@zaki50 I have problem when I use custom name of the database. 

```
Realm.init(this);
RealmConfiguration config = new RealmConfiguration.Builder()
                .name(Constants.REALM_DATABASE_NAME)
                .schemaVersion(4)
                .migration(new MyMigration())
                .build(); 
```

Stetho show two database: default and my, but both are empty (don't show tabels structure). 

![zrzut ekranu 2016-10-05 o 13 27 52 1](https://cloud.githubusercontent.com/assets/14806683/19111664/1d6318f2-8b01-11e6-89b0-48e75e3186c2.png)
__

When I remove name() method form my config everything is ok. I can preview my database.
I use:
stethoVersion                       = '1.4.1'
realmStethoVersion             = '2.0.0'
realm                                     = '2.0.1'

**Comment by flyilxm** [comment]
In the 'RealmPeerManager' class, line 129, method 'openSharedRealm', the param 'databaseId' didn't put in the builder.

**Comment by zaki50** [comment]
thank you for reporting.

I'll investigate it.

**Comment by wangzs** [comment]
encount the same problem. when i change version[realm-gradle-plugin:1.1.1  & com.facebook.stetho:stetho:1.3.1 & com.uphyca:stetho_realm:0.9.0] it can work.

problem version[io.realm:realm-gradle-plugin:2.0.2 & com.facebook.stetho:stetho:1.4.1 & com.uphyca:stetho_realm:2.0.0]

**Comment by cknibbe** [comment]
+1

**Comment by whenasmile** [comment]
+1

**Comment by Sejen27** [comment]
still not fixed

**Comment by jackleeforce** [comment]
i'm encount the same problem,and i found when i write some object to default.realm with Code:Realm.getDefaultInstance(),
everything is fine include my custom realm name.

I use:
stethoVersion = '1.4.1'
realmStethoVersion = '2.0.0'
realm = '2.2.1'

**Comment by arnereichenberg** [comment]
+1

**Comment by luckybk93** [comment]
To solove this problem.

Uninstall your app.
Remove .name("yourname.realm") in RealmConfiguration.Builder()
Re install.
My configuration:
AppController:
Realm.init(this);
byte[] key = new byte[64];
RealmConfiguration config = new RealmConfiguration.Builder()
// .name("yourname.realm")
// .encryptionKey(key)
.schemaVersion(4)
.deleteRealmIfMigrationNeeded()
.modules(Realm.getDefaultModule())
.build();
// Use the config
Realm.setDefaultConfiguration(config);
Stetho.initialize(
Stetho.newInitializerBuilder(this)
.enableDumpapp(Stetho.defaultDumperPluginsProvider(this))
.enableWebKitInspector(RealmInspectorModulesProvider.builder(this).build())
.build());
Gradle
-> maven {url 'https://github.com/uPhyca/stetho-realm/raw/master/maven-repo'}
-> compile 'com.facebook.stetho:stetho:1.4.2'
compile 'com.uphyca:stetho_realm:2.0.0'

**Comment by rajendratechie** [comment]
I am also facing the same issue with the custom name of the database.
I use:
debugCompile 'com.facebook.stetho:stetho:1.4.2'
    debugCompile 'com.uphyca:stetho_realm:2.0.0'
 maven {
        url 'https://github.com/uPhyca/stetho-realm/raw/master/maven-repo'
    }

Did anyone fix the issue?

**Comment by madaline** [comment]
stetho-realm 2.0.1 is released.

edit build.gradle:

repositories {
    maven { url 'https://github.com/linchaolong/stetho-realm/raw/master/maven-repo' }
}

dependencies {
    compile 'com.facebook.stetho:stetho:1.4.2'
    compile 'com.uphyca:stetho_realm:2.0.1'
}

**Comment by rajendratechie** [comment]
Thanks @madaline, it's working now !!!

**Comment by zaki50** [comment]
I've released 2.1.0 that contains the fix.
I'll close this issue.
